sdk/eventhubs/azure-messaging-eventhubs/test/eventhubs-stress-test/src/scenarios/inc/shared_functions.hpp

```diff
@@ -14,12 +14,25 @@
 #include <memory>
 #include <utility>
 
+#if defined(_MSC_VER)
+// The OpenTelemetry headers generate a couple of warnings on MSVC in the OTel 1.2 package, suppress
+// the warnings across the includes.
+#pragma warning(push)
+#pragma warning(disable : 4100)
+#pragma warning(disable : 4244)
+#pragma warning(disable : 6323) // Disable "Use of arithmetic operator on Boolean type" warning.
+#endif
```

Comment:
Do we know of any other places that need these suppression?

Going forward, how do we reduce chances of re-introducing such errors, when new files get added? My assumption is all places that include `opentelemetry` headers need to have these warning guards aroudn them.

Reply:
Just did a quick search for `#include <opentelemetry/`, looks like only this file had these warning suppressions missing.

Reply:
If there were, they wouldn't have passed the CI pipeline.